pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -324,6 +325,7 @@ self: super: {
   http-link-header = dontCheck super.http-link-header; # non deterministic failure https://hydra.nixos.org/build/75041105
   ihaskell = dontCheck super.ihaskell;
   influxdb = dontCheck super.influxdb;
+  irc-core = doJailbreak super.irc-core;
```

Comment:
Could you report this upstream and link to the issue here in the code?

This is so that we have some way to track when the `doJailbreak` can be removed.

Reply:
I can and am about to start making them. I have a problem in that I'm really not sure what I'm supposed to say; I could really use some guidance or a template or even another issue someone's made that's a good example to follow (there's probably another time I've asked that you explained it I could link to I will try to find) I appreciate all the work you do. [I will paint you any stickers you want.](https://mostlyabsurd.com/files/stickers-for-all.html) I'm on the IRC and Discord. let me know what you'd like I'm still making them... I think I tried getting your attention on Twitter a while ago by adding you to a group but I don't know if it worked :laughing: . and all of [my contact information are here](https://www.mostlyabsurd.com/contact/#socialize-with-me) Anyway, as far as the issues I'm supposed to make go... what I want to say is far too blunt and that's just how I am. (I really don't *want* to nor *will* I say this but it's how I think a lot of people unfamiliar with NixOS experience it. I do know a few people that are bewildered by it though even after I try and explain, which is probably me not being good at explaining :stuck_out_tongue:  . anyway that's why I could use help. >.>

>  "we use stackage on NixOS (@cdepillabout and I know part of the reason we do is there are too many versions on hackage for NixOS to have them all. doesn't make it any less annoying) and you publish to hackage and there's probably a reason you don't use stackage I know a lot of people don't like it (I was already not a fan before I even heard of NixOS and I've only come to dislike it more) but I am here to suggest you do anyway for all 2 (idk how many people use it I hope there are more) people that use glirc on NixOS who don't want to use overlays so I'm cluttering your issue tracker; also please update your version bounds ASAP because while you don't use NixOS and only support cabal install where what you have works fine the fact that this doesn't work gives a bad impression about Nix"

 @glguy is a friend and he's pretty quick about fixing the bounds if I ask (sometimes he's even started before I ask). but most of these people I don't know at all :frowning_face: and what happens if they don't want to or can't? can they just be like permanent? and If they need updated to keep working it's not like I'm already updating them to keep them working >.> they're there most of the time anyway.

Reply:
[found your comment](https://github.com/NixOS/nixpkgs/pull/92618#discussion_r451303303)

btw here's one of the haskell stickers that I painted
![](https://mostlyabsurd.com/pieces/images/2019-12-11-trans-haskell-1.jpg)
and NixOS
![](https://user-images.githubusercontent.com/35715/100150004-e2cd2780-2e96-11eb-8854-c18163dc4f05.png)
I'm giving like at least 144 away (but so far there are a lot more stickers than there are people to give them...) :cry:  (they're really cool and take a long time and, well, they aren't cheap lol. but people really like them) Everyone here does so much good (unpaid, often underappreciated) work that it's the least I can do (it's all explained in the blog...)

@peti you can have some too even though you seem to make most of the commits that make me fix them :stuck_out_tongue_winking_eye: but I don't think it'd work much at all without you so you have that in your favor :wink: 

really anyone that wants one just lmk